docs/src/main/sphinx/security/https.rst

```diff
@@ -0,0 +1,357 @@
+.. _https_tls_top:
+
+=============
+HTTPS and TLS
+=============
+
+Trino (formerly PrestoSQL) runs with no security configuration in its default
```

Comment:
remove the formerly PrestoSQL ... we dont use that anywhere in the docs at this stage.

Reply:
Done